### PR TITLE
SEO-176113-React-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the missing H1

Regards,
Beryl.

### CODE REVIEW

File: ReactJS/SunburstChart/Tooltip.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Tooltip
-description: Learn how to add Tooltip to Sunburstchart .
+title: Add Tooltip to Sunburst Chart | Syncfusion
```
Comment (Yvone-Atieno) [inline ReactJS/SunburstChart/Tooltip.md:3]: Tooltip in ReactJS Sunburst Chart Control | Syncfusion

File: ReactJS/SunburstChart/Tooltip.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+description: Learn here about how to how to add Tooltip to Sunburst Chart in Syncfusion Essential ReactJS Control, its elements, and more.
 platform: ts
 control: Sunburst Chart
 documentation: ug
 ---
 
-# Tooltip  
+# Adding Tooltip to Sunburst Chart in ReactJS
```
Comment (Yvone-Atieno) [inline ReactJS/SunburstChart/Tooltip.md:10]: update proper h1

File: ReactJS/SunburstChart/Tooltip.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ ReactDOM.render(
 
 {% endhighlight %}
 
-![](/js/SunburstChart/Tooltip_images/Tooltip_img2.png)
+![Tooltip Template](/js/SunburstChart/Tooltip_images/Tooltip_img2.png)
```
Comment (Yvone-Atieno) [inline ReactJS/SunburstChart/Tooltip.md:65]: Update image alt properly

File: ReactJS/SunburstChart/Tooltip.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ documentation: ug
 
 {% endhighlight %}
 
-![](/js/SunburstChart/Tooltip_images/Tooltip_img1.png)
+![Adding Tooltip to Sunburst Chart](/js/SunburstChart/Tooltip_images/Tooltip_img1.png)
```
Comment (Yvone-Atieno) [inline ReactJS/SunburstChart/Tooltip.md:30]: Update image alt properly

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ReactJS/SunburstChart/Tooltip.md (hunk 1 above)
Comment (berylouma) [reply ReactJS/SunburstChart/Tooltip.md:3]: Added

File: ReactJS/SunburstChart/Tooltip.md (hunk 2 above)
Comment (berylouma) [reply ReactJS/SunburstChart/Tooltip.md:10]: Updated

File: ReactJS/SunburstChart/Tooltip.md (hunk 3 above)
Comment (berylouma) [reply ReactJS/SunburstChart/Tooltip.md:65]: Updated

File: ReactJS/SunburstChart/Tooltip.md (hunk 4 above)
Comment (berylouma) [reply ReactJS/SunburstChart/Tooltip.md:30]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**